### PR TITLE
[zh] Update terms about load-balance, observability and _index page

### PR DESCRIPTION
**Pull request by windsonsea**
Update terms to keep consistent with our [homepage](https://istio.io/latest/zh/):

- Load balance：负载平衡 -> 负载均衡
- Observability：可观察性 -> 可观测性

Also sync the homepage with en version. See [preview](https://deploy-preview-14306--preliminary-istio.netlify.app/latest/zh).

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/blog/2021/zero-config-istio/index.md [hunk 1]
```diff
@@ -21,13 +21,13 @@ Istio 自动为网格中 pod 之间的流量启用 [mutual TLS](/zh/docs/concept
...
-## 可观察性 {#observability}
+## 可观测性 {#observability}
 
-Istio 自动为网格中的所有服务通信生成详细的遥测数据。这种遥测提供了服务行为的可观察性，使运营商能够对其应用程序进行故障排除、维护和优化——而不会给服务开发人员带来任何额外的负担。通过 Istio，运维人员可以全面了解受监控的服务如何与其他服务以及 Istio 组件本身进行交互。
+Istio 自动为网格中的所有服务通信生成详细的遥测数据。这种遥测提供了服务行为的可观测性，使运营商能够对其应用程序进行故障排除、维护和优化——而不会给服务开发人员带来任何额外的负担。通过 Istio，运维人员可以全面了解受监控的服务如何与其他服务以及 Istio 组件本身进行交互。
 
 所有这些功能都是由 Istio 添加的，无需任何配置。 也可以与 Prometheus, Grafana, Jaeger, Zipkin, and Kiali 等工具 [集成（Integrations）](/zh/docs/ops/integrations/) 使用。
 
-有关 Istio 提供的可观察性的更多信息，请查看 [可观察性概述（observability overview）](/zh/docs/concepts/observability/)。
+有关 Istio 提供的可观测性的更多信息，请查看 [可观测性概述（observability overview）](/zh/docs/concepts/observability/)。
```
Comment (my-git9) [inline content/zh/blog/2021/zero-config-istio/index.md:30]:
```suggestion
有关 Istio 提供的可观测性的更多信息，请查看[可观测性概述（observability overview）](/zh/docs/concepts/observability/)。
```

File: content/zh/about/service-mesh/index.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ doc_type: about
 服务网格不仅描述了这个网络，而且还描述了分布式应用程序组件之间的交互。所有在服务之间传递的数据都由服务网格控制和路由。
 
 随着分布式服务的部署——比如基于 Kubernetes 的系统——规模和复杂性的增长，它可能会变得更加难以理解和管理。
-需求可以包括发现、负载平衡、故障恢复、度量和监视。微服务体系结构通常还有更复杂的操作需求，比如 A/B
+需求可以包括发现、负载均衡、故障恢复、度量和监视。微服务体系结构通常还有更复杂的操作需求，比如 A/B
 测试、canary 部署、速率限制、访问控制、加密和端到端身份验证。
```
Comment (wilsonwu) [inline content/zh/about/service-mesh/index.md:33]:
```suggestion
测试、金丝雀部署、速率限制、访问控制、加密和端到端身份验证。
```

File: content/zh/blog/2021/zero-config-istio/index.md [hunk 3]
```diff
@@ -21,13 +21,13 @@ Istio 自动为网格中 pod 之间的流量启用 [mutual TLS](/zh/docs/concept
 
 熟悉自动 TLS 后，您可以选择 [仅允许 mTLS 流量（allow only mTLS traffic）](/zh/docs/tasks/security/authentication/mtls-migration/)，或配置自定义 [授权策略（authorization policies）](/zh/docs/tasks/security/authorization/) 满足您的需求。
 
-## 可观察性 {#observability}
+## 可观测性 {#observability}
 
-Istio 自动为网格中的所有服务通信生成详细的遥测数据。这种遥测提供了服务行为的可观察性，使运营商能够对其应用程序进行故障排除、维护和优化——而不会给服务开发人员带来任何额外的负担。通过 Istio，运维人员可以全面了解受监控的服务如何与其他服务以及 Istio 组件本身进行交互。
+Istio 自动为网格中的所有服务通信生成详细的遥测数据。这种遥测提供了服务行为的可观测性，使运营商能够对其应用程序进行故障排除、维护和优化——而不会给服务开发人员带来任何额外的负担。通过 Istio，运维人员可以全面了解受监控的服务如何与其他服务以及 Istio 组件本身进行交互。
 
 所有这些功能都是由 Istio 添加的，无需任何配置。 也可以与 Prometheus, Grafana, Jaeger, Zipkin, and Kiali 等工具 [集成（Integrations）](/zh/docs/ops/integrations/) 使用。
```
Comment (wilsonwu) [inline content/zh/blog/2021/zero-config-istio/index.md:28]:
```suggestion
所有这些功能都是由 Istio 添加的，无需任何配置。也可以与 Prometheus、Grafana、Jaeger、Zipkin 和 Kiali 等工具[集成（Integrations）](/zh/docs/ops/integrations/)使用。
```

File: content/zh/blog/2021/zero-config-istio/index.md [hunk 4]
```diff
@@ -64,6 +64,6 @@ Hostname=echo-cb96f8d94-rpfqz
 Hostname=echo-cb96f8d94-cgmxr
 {{< /text >}}
 
-在这里，我们可以看到我们的请求是 [轮询（round-robin）](/zh/docs/concepts/traffic-management#load-balancing-options) 在所有后端之间进行负载平衡的。
+在这里，我们可以看到我们的请求是 [轮询（round-robin）](/zh/docs/concepts/traffic-management#load-balancing-options) 在所有后端之间进行负载均衡的。
```
Comment (wilsonwu) [inline content/zh/blog/2021/zero-config-istio/index.md:67]:
```suggestion
在这里，我们可以看到我们的请求是[轮询（round-robin）](/zh/docs/concepts/traffic-management#load-balancing-options)在所有后端之间进行负载均衡的。
```

File: content/zh/blog/2021/zero-config-istio/index.md [hunk 5]
```diff
@@ -64,6 +64,6 @@ Hostname=echo-cb96f8d94-rpfqz
...
 
 除了这些默认设置之外，Istio 还提供了 [各种流量管理设置（variety of traffic management settings）](/zh/docs/concepts/traffic-management/) 的自定义设置，包括超时、重试等等。
```
Comment (wilsonwu) [inline content/zh/blog/2021/zero-config-istio/index.md:69]:
```suggestion
除了这些默认设置之外，Istio 还提供了[各种流量管理设置（variety of traffic management settings）](/zh/docs/concepts/traffic-management/)的自定义设置，包括超时、重试等等。
```

File: content/zh/blog/2022/merbridge/index.md [hunk 6]
```diff
@@ -5,11 +5,11 @@ publishdate: 2022-03-07
...
 keywords: [Istio,ebpf,iptables,sidecar]
 ---
-Istio 在流量治理、加密、可观察性和策略方面的能力的秘密都在 Envoy 代理中。Istio 使用 Envoy 作为 “sidecar” 来拦截服务流量，并使用 iptables 配置的内核 `netfilter` 包过滤功能。
+Istio 在流量治理、加密、可观测性和策略方面的能力的秘密都在 Envoy 代理中。Istio 使用 Envoy 作为 “sidecar” 来拦截服务流量，并使用 iptables 配置的内核 `netfilter` 包过滤功能。
 
 使用 iptables 来执行这种拦截有一些缺点。由于 netfilter 是一种高度通用的数据包过滤工具，在到达目的套接字之前，需要应用多种路由规则和数据过滤过程。例如，从网络层到传输层，netfilter 会使用预定义的规则进行多次处理，如 `pre_routing`，`post_routing` 等。当报文变成 TCP 或 UDP 报文，并被转发到用户空间时，还需要执行一些额外的步骤，如报文验证、协议策略处理和目的套接字搜索。当将 sidecar 配置为拦截流量时，原始数据路径可能会变得非常长，因为重复的步骤会执行多次。
 
-在过去的两年中，[eBPF](https://ebpf.io/)已经成为一种技术趋势，许多基于 eBPF 的项目已经发布到社区。像[Cilium](https://cilium.io/)和[Pixie](http://px.dev)这样的工具展示了 eBPF 在可观察性和网络数据包处理方面的大量用例。通过 eBPF 的 `sockops` 和 `redir` 功能，可以通过直接从入口 socket 传输到出口 socket 来有效地处理数据包。在 Istio mesh 中，可以使用 eBPF 来替代 iptables 规则，并通过缩短数据路径来加速数据平面。
+在过去的两年中，[eBPF](https://ebpf.io/)已经成为一种技术趋势，许多基于 eBPF 的项目已经发布到社区。像[Cilium](https://cilium.io/)和[Pixie](http://px.dev)这样的工具展示了 eBPF 在可观测性和网络数据包处理方面的大量用例。通过 eBPF 的 `sockops` 和 `redir` 功能，可以通过直接从入口 socket 传输到出口 socket 来有效地处理数据包。在 Istio mesh 中，可以使用 eBPF 来替代 iptables 规则，并通过缩短数据路径来加速数据平面。
```
Comment (wilsonwu) [inline content/zh/blog/2022/merbridge/index.md:12]:
```suggestion
在过去的两年中，[eBPF](https://ebpf.io/) 已经成为一种技术趋势，许多基于 eBPF 的项目已经发布到社区。像 [Cilium](https://cilium.io/)和[Pixie](http://px.dev) 这样的工具展示了 eBPF 在可观测性和网络数据包处理方面的大量用例。通过 eBPF 的 `sockops` 和 `redir` 功能，可以通过直接从入口 socket 传输到出口 socket 来有效地处理数据包。在 Istio 网格中，可以使用 eBPF 来替代 iptables 规则，并通过缩短数据路径来加速数据平面。
```

**Review by wilsonwu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**